### PR TITLE
Enhance/subscription image count

### PR DESCRIPTION
**Pull request by zamandevs**
added filters & actions for gallery image restriction.

### CODE REVIEW

File: includes/wc-functions.php [hunk 1]
```diff
@@ -32,6 +32,8 @@ function dokan_process_product_meta( $post_id, $data = [] ) {
 
     // Gallery Images
     if ( isset( $data['product_image_gallery'] ) ) {
+        $data = apply_filters( 'restrict_product_image_gallery_on_edit', $data );
```
Comment (sabbir1991) [inline includes/wc-functions.php:35]: Please add `dokan` prefix before filter name.

File: templates/products/tmpl-add-product-popup.php [hunk 2]
```diff
@@ -177,3 +175,4 @@
         </form>
     </div>
 </script>
+<?php do_action( 'dokan_product_gallery_image_count' );?>
```
Comment (sabbir1991) [inline templates/products/tmpl-add-product-popup.php:178]: Please change the action name `dokan_product_gallery_image_count ` to `dokan_add_product_js_template_end`

**Comment by alamgircsebd** [comment]
Need to fix conflicts errors

**Comment by nurul-umbhiya** [comment]
@zamandevs I'm closing this pr now due to merge conflict, kindly make a new PR with your changes

**Pull request closed without merging**